app/src/main/java/com/davidtomas/taskyapp/features/agenda/presentation/AgendaDetail/AgendaDetailScreen.kt

```diff
@@ -0,0 +1,265 @@
+package com.davidtomas.taskyapp.features.agenda.presentation.agendaDetail
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material3.Button
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.tooling.preview.PreviewParameter
+import androidx.compose.ui.tooling.preview.PreviewParameterProvider
+import androidx.compose.ui.unit.dp
+import com.davidtomas.taskyapp.R
+import com.davidtomas.taskyapp.core.presentation.components.Header
+import com.davidtomas.taskyapp.coreUi.TaskyAppTheme
+import com.davidtomas.taskyapp.features.agenda.domain.model.AgendaType
+import com.davidtomas.taskyapp.features.agenda.presentation.agendaDetail.components.AgendaTypeComposable
+import com.davidtomas.taskyapp.features.agenda.presentation.agendaDetail.components.AttendeeItem
+import com.davidtomas.taskyapp.features.agenda.presentation.agendaDetail.components.DescriptionComposable
+import com.davidtomas.taskyapp.features.agenda.presentation.agendaDetail.components.NotificationTimePicker
+import com.davidtomas.taskyapp.features.agenda.presentation.agendaDetail.components.PhotosComposable
+import com.davidtomas.taskyapp.features.agenda.presentation.agendaDetail.components.PillsSelector
+import com.davidtomas.taskyapp.features.agenda.presentation.agendaDetail.components.TimeDatePicker
+import com.davidtomas.taskyapp.features.agenda.presentation.agendaDetail.components.TitleComposable
+
+@Composable
+fun EventDetailScreen(
+    agendaType: AgendaType,
```

Comment:
I think you forgot to rename this composable, otherwise the agenda type doesn't make sense if it's always an event